### DISCUSSION TITLE
Cant publish config

### DISCUSSION THREAD

**Discussion by LvckyAPI** [dedoc/scramble#614] [Q&A]
I tried to publish the config like in docs (https://scramble.dedoc.co/installation) 

But it does not work

```bash
~/PhpstormProjects/marina-api-laravel git:[4-get-discord-user-avatars-docs]
pa vendor:publish --provider="Dedoc\Scramble\ScrambleServiceProvider" --tag="scramble-config"

   INFO  No publishable resources for tag [scramble-config].  
```

**Comment by romalytvynenko** [comment]
@LvckyAPI 

Cannot reproduce. Something is off with your setup. 

<img width="1191" alt="image" src="https://github.com/user-attachments/assets/247a1a8b-6b3c-4dfb-944f-9eae06eb60a9">
